client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Remove the default generated Readme by create react app, and add your capstone component details in Readme, along with installation guild.